modules/vaos/spec/services/systems_service_spec.rb

```diff
@@ -110,4 +110,25 @@
       end
     end
   end
+
+  describe '#get_system_facilities' do
+    context 'with a 200 response' do
+      it 'returns an array of size 6' do
```

Comment:
What does 6 mean here? A translation of the magic number might help someone understand in the future.

Reply:
The array returned in the VCR cassette is 6. We're just verifying the response returned from downstream here. There isn't really anything magic about it. Could have just as easily verified that `is_an_instance_of(Array)` but this is a bit more detailed.